src/ucp/tag/offload/eager.c

```diff
@@ -128,13 +125,35 @@ static ucs_status_t ucp_proto_eager_tag_offload_bcopy_init(
     };
 
     /* offload proto can not be used if no tag offload lane configured */
-    if (!ucp_proto_eager_check_op_id(init_params, 1)) {
+    if (!ucp_proto_eager_check_op_id(init_params, op_id, 1)) {
         return UCS_ERR_UNSUPPORTED;
     }
 
     return ucp_proto_single_init(&params);
 }
 
+
+static ucs_status_t
+ucp_proto_eager_tag_offload_bcopy_progress(uct_pending_req_t *self)
+{
+    ucp_request_t *req = ucs_container_of(self, ucp_request_t, send.uct);
+    const ucp_proto_single_priv_t *spriv = req->send.proto_config->priv;
```

Comment:
unaligned '='?

Reply:
+ declare `const` first?

Reply:
- regarding alignment, it looks better this way. Imo, looks too ugly if aligned
- why const has to be the first one?

Reply:
it can't be first because it depends on req